static/playHLSVideo.js

```diff
@@ -30,12 +30,21 @@
 
             function initializeHls() {
                 newVideo.removeEventListener('play', initializeHls);
-
                 var hls = new Hls({ autoStartLoad: false });
                 hls.loadSource(playlist);
                 hls.attachMedia(newVideo);
                 hls.on(Hls.Events.MANIFEST_PARSED, function () {
-                    hls.loadLevel = hls.levels.length - 1;
+                    hls.loadLevel = 0;
```

Comment:
I assume it is something to do with this - this code sets it to the lowest (it's sorted by bitrate by Reddit) but it used to be the highest.

Reply:
I know that this sets it to the lowest. I wanted to ask how/where do I add some preference button in settings that lets you set some preference.

Reply:
Ah, I see. I would just select the highest by default as that was the previous behavior. Now there's the chance to reduce that at least. But I don't think we really need to configure this unless there's a ton of demand for it (especially given there's so many different resolutions and bandwidths for videos Reddit returns)

Reply:
Ok np. But still for my knowledge can you tell me how can I get any preference value say `post_sort` in `playHlsVideo.js`.

Reply:
Also should I revert to highest?

Reply:
I think that would be a good idea just to retain previous behavior, yeah.